sphinx/ext/coverage.py

```diff
@@ -121,6 +124,14 @@ def write_c_coverage(self) -> None:
                 write_header(op, filename)
                 for typ, name in sorted(undoc):
                     op.write(' * %-50s [%9s]\n' % (name, typ))
+                    if self.config.coverage_print_missing_c_items:
+                        if self.app.quiet or self.app.warningiserror:
```

Comment:
I think the runtime option `quiet` is stronger than a configuration. So it would be better to suppress these messages.

Reply:
I copied this behavior from `linkcheck` [here](https://github.com/sphinx-doc/sphinx/blob/3.x/sphinx/builders/linkcheck.py#L258-L262).
I wrapped the part with `if self.config.coverage_show_missing_items:` because I wanted to keep the default behavior which is nothing gets printed/logged.

How do you @tk0miya want me to change the behavior?
When should:
1. nothing be logged?
2. be logged at `info` level?
3. be logged at `warning` level?

Current behavior:
1. When `coverage_show_missing_items=False` (default and independent on other settings)
2. When `coverage_show_missing_items=True` **and neither** `-q` **nor** `-W` 
3. When `coverage_show_missing_items=True` **and** `-q` **and/or** `-W`

Reply:
I thought nothing logged is better at first. But it is better to do the same as linkcheck does. Okay, keep as is.